### DISCUSSION TITLE
Jupyter notebook is not detecting module and hover functionality is not working properly

### DISCUSSION THREAD

**Discussion by Ritesh313** [microsoft/vscode-jupyter#15263] [Questions and Answers]
Hi, I'm using VS code insiders and facing issue with what I think is because of Pylance or Ruff? I'm not entirely sure what's happening, hence posting it here instead of issues. I'm facing following issues:
1. Some of the module names don't change color (see screenshot module time, part of warnings)
2. At times, it'll start throwing random error for content that's not even in the same cell (and doesn't have any error most times)
![image](https://github.com/microsoft/vscode-jupyter/assets/36135489/59edf427-7498-45f0-ab37-29fa7e39af9a)

I'm using:
VScode insiders 1.87.0 (I'm using remote tunnel on a VScode insiders on windows machine to remote to a linux server)
Jupyter v2024.1.1
Intellicode v1.2.30
IntelliCode API Usage Examples v0.2.8
Pylance v2024.2.104 (pre-release)
Python v2024.3.10581547 (pre-release)
Ruff v2024.12.0

Please let me know if I should provide any other information. Thanks in advance!

**Comment by DonJayamanne** [comment]
Thanks for bringing this to our attention and sorry you are running into this.
Please uninstall one or both of ruff/pylance and identify the extnesion that causes the issue.
Then please file an issue against the relevant repo.
